src/Accordion.js

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { elementType } from 'prop-types-extra';
+import { useBootstrapPrefix } from './ThemeProvider';
+import AccordionContext from './AccordionContext';
+import AccordionToggle from './AccordionToggle';
+import SelectableContext from './SelectableContext';
+import AccordionCollapse from './AccordionCollapse';
+
+const propTypes = {
+  /** Set a custom element for this component. */
+  as: elementType,
+
+  /** @default 'accordion' */
+  bsPrefix: PropTypes.string,
+};
+
+const defaultProps = {
+  as: 'div',
+};
+
+const Accordion = React.forwardRef(
+  ({ as: Component, bsPrefix, children, className, ...props }, ref) => {
+    bsPrefix = useBootstrapPrefix(bsPrefix, 'accordion');
+
+    const [openedId, setOpenedId] = useState('0');
```

Comment:
should we use `uncontrollable` here and have something like `defaultActiveKey`? we should have some capacity for the accordion to start in a "fully collapsed" state

Reply:
I ended up using `useUncontrolled` for this, and updated the docs to include an example of how to set Accordion to a "fully collapsed" state